tests/templates/test_state_preparations/test_state_prep_mps.py

```diff
@@ -21,232 +21,1581 @@
+                            0.08752316,
+                            0.09132769,
+                            0.0295163,
+                            0.06551486,
+                            0.07988889,
+                            0.10570001,
+                            0.0758292,
+                            0.08473836,
+                            0.0672397,
+                            0.02405432,
+                            0.0972505,
+                            0.09033668,
+                            0.03338503,
+                            0.00997671,
+                            0.08601812,
+                            0.08988569,
+                            0.01974867,
+                            0.07537103,
+                            0.044631,
+                            0.01163534,
+                            0.07778555,
+                            0.00500877,
+                            0.05604735,
+                            0.06193424,
+                            0.026859,
+                            0.05198305,
+                            0.05335228,
+                            0.06338206,
+                            0.07981056,
+                            0.03128861,
+                            0.00411518,
+                            0.00500641,
+                            0.08231421,
+                            0.0835383,
+                            0.02039181,
+                            0.02134244,
+                            0.08108945,
+                            0.06373903,
+                            0.05857015,
+                            0.06034281,
+                            0.08319761,
+                            0.07588063,
+                            0.0979415,
+                            0.08503184,
+                            0.05053136,
+                            0.04204917,
+                            0.00650433,
+                            0.07961832,
+                            0.06016469,
+                            0.04312555,
+                            0.02963368,
+                            0.09109303,
+                            0.01501264,
+                            0.0021724,
+                            0.08730843,
+                            0.07497851,
+                            0.08032175,
+                            0.06144865,
+                            0.07071516,
+                            0.0361662,
+                            0.0044735,
+                            0.01776762,
+                            0.04240971,
+                            0.06001864,
+                            0.03403537,
+                            0.01691472,
+                            0.0546236,
+                        ]
+                    )
+                ],
+                8, 4
+            ),
+        ],
+    )
+    def test_correctness(self, mps, state, num_wires, num_work_wires):
+        """Test correctness of the solution
```

Comment:
What is the solution here? What are we aiming to test?
Is this system test something we can avoid using a QNode for and perform as a unit/integration test instead (see https://github.com/PennyLaneAI/guidance-docs/blob/master/development/testing-guidelines.md#writing-unit-tests for further details)

Reply:
Added more info in the docstring. In this case we have to build the qnode and run the circuit to check the `qml.state` that is generated

Reply:
Shouldn't this be possible to test without creating a QNode?

Reply:
Updated using QuantumScript